src/sentry/migrations/0348_issue_alert_fallback.py

```diff
@@ -0,0 +1,84 @@
+# Generated by Django 2.2.28 on 2023-01-04 19:14
+
+import logging
+
+from django.db import migrations, transaction
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def set_issue_alert_fallback(rule, fallthrough_choice):
+    actions = rule.data.get("actions", [])
+    new_actions = []
+    for action in actions:
+        id = action.get("id")
+        target_type = action.get("targetType")
+        if id == "sentry.mail.actions.NotifyEmailAction" and target_type == "IssueOwners":
+            action.update({"fallthroughType": fallthrough_choice})
+            new_actions.append(action)
+
+    rule.save()
```

Comment:
What happens if there are no actions for the rule?
Do we just save it but it does nothing? (since we are not changing anything)

Reply:
That's a good call - refactored this to only save if we make changes